### DISCUSSION TITLE
Problem recognizing the language of repositories

### DISCUSSION THREAD

**Discussion by mmdbalkhi** [community/community#4727] [Other Feature Feedback, Questions, & Ideas]
# Problem recognizing the language of repositories

Hello. I have a repository in which I have used two languages, but I have only seen one of the Github languages.

![repo](https://user-images.githubusercontent.com/65954744/126357722-eef8fd70-37b4-42d2-a31d-0bb8cf6d96eb.png)
![onefetch](https://user-images.githubusercontent.com/65954744/126357744-9e3ee26f-d740-404b-944f-b47245ccd86c.png)
link([+](https://github.com/KomeilParseh/learning-rust))

As you can see, [onefetch](https://github.com/o2sh/onefetch) is correct, but GitHub only shows rust and not C!

The problem sometimes becomes that it shows additional languages that were not used at all in that repository!

![pyabr](https://user-images.githubusercontent.com/65954744/126358452-4aea9657-2f6c-4f52-8da7-a2a20060bfbb.png)
link([+](https://github.com/PyFarsi/pyabr))

**Comment by danBamikiya** [comment]
Hi, concerning the first repo, I think its has to do with the way they 'calculate' languages percent/ratio. Because if you click `Rust 100.0%` you'll see all the other languages  used: 
![image](https://user-images.githubusercontent.com/58262528/129616034-ea1c6816-eb8e-40bf-9d9c-6ba8566ca205.png)
 Also  concerning the second repo, HTML, Java, Shell and C# have been used (click on the individual languages to see the corresponding files) again it has to do with the way they calculate languages percent/ratio.

I'll also like to petition to Github product folks to consider increasing the number of languages shown to 9. CC @evi-liu
